src/services/request.service.ts

```
@@ -0,0 +1,30 @@
+    requestData: CreateRequestDTO,
+  ): Promise<RequestResponseDTO> {
+    try {
+      const result = await this.requestRepository.createRequest({
+        id: uuidv4(),
+        ...requestData,
+        status: "Pending",
+      });
+
+      return result;
+    } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)